setup.py

```diff
@@ -58,6 +58,6 @@ def get_version(project_dir=os.curdir):
 if __name__ == '__main__':
     os.environ['PBR_VERSION'] = get_version()
     setup(
-        setup_requires=['pbr'],
+        setup_requires=['pbr', 'dulwich'],
```

Comment:
OK, so regardless if `pbr` pulls it or not, we have an explicit import in the version_manager.py, so I'll keep it.